contracts/BootstrapRewards.sol

```diff
@@ -39,6 +39,7 @@ contract BootstrapRewards is IBootstrapRewards, Ownable {
         require(address(_externalToken) != address(0), "externalToken must not be 0");
 
         externalToken = _externalToken;
+        // TODO - The initial lastPayedAt should be set in the first assignRewards.
```

Comment:
Which time period will the first call to assignRewards will use?

Reply:
none / lastPayedAt = now.
The first call will init it, as the contracts are likely to be deployed prior to enablement

Reply:
So maybe a separate enablement method?

Reply:
(so enablement can only be done by governance and not by everyone)